plecos/samples/algorithm_metadata_remote.json

```diff
@@ -6,7 +6,6 @@
     "datePublished": "2019-02-08T08:13:49Z",
     "license": "CC-BY",
     "name": "My super algorithm",
-    "price": "1",
```

Comment:
if not price, is it a `dtCost` ?

Reply:
plecos only does metadata section, and not the services section.  there is no reason to have a dtCost for metadata